components/slack/actions/send-direct-message/send-direct-message.js

```diff
@@ -0,0 +1,30 @@
+const slack = require('../../slack.app.js')
+const { WebClient } = require('@slack/web-api')
+
+module.exports = {  
+  key: "slack-send-direct-message",
+  name: "Send a Direct Message",
+  description: "Send a direct message to a single user.",
```

Comment:
I'm curious, did you test this out?  Casual reading of the slack API makes it sound like you can't directly message between 2 users even with the as_user set.